tests/test_file_ops.py

```diff
@@ -0,0 +1,66 @@
+import shutil
+import os
+from pathlib import Path
+import sys
+sys.path.insert(1, "nesta_ds_utils/")
+import file_ops
+import pytest
```

Comment:
```suggestion
import shutil
import os
from pathlib import Path
import sys
import pytest

sys.path.insert(1, "nesta_ds_utils/")
import file_ops
```

Small but helps to identify the module being tested.